### PR TITLE
Switch to manual sync intervals

### PR DESCRIPTION
**Pull request by ericnewcomer**

### CODE REVIEW

**Review by nicpottier — APPROVED** [review]

**Review by norkans7 — APPROVED** [review]

**Comment by nicpottier** [comment]
@ericnewcomer do you happen to remember the goal of this one or why we didn't merge it back then?

**Comment by ericnewcomer** [comment]
I believe we previously synced manually. This was just an update to get us current with the ever-changing recommended Android way to accomplish this.

**Comment by nicpottier** [comment]
@norkans7 can you merge the other PR in, then merge master into this and give it some on device testing love?

**Comment by norkans7** [comment]
Sure

File: app/src/main/java/io/rapidpro/androidchannel/SyncHelper.java [hunk 1]
```diff
@@ -51,23 +35,12 @@
...
-     */
-    static void enqueueWork(Context context, Intent work) {
-        enqueueWork(context, SyncService.class, JOB_ID, work);
-    }
+import static android.content.Context.CONNECTIVITY_SERVICE;
 
+public class SyncHelper {
+
+    // how many seconds between polling
+    public static final long POLLING_INTERVAL = 60;
```
Comment (norkans7) [inline app/src/main/java/io/rapidpro/androidchannel/SyncHelper.java]: I think 60 secs is a little aggressive, what about 5 min or 10 min? @ericnewcomer @nicpottier

**Comment by ericnewcomer** [comment]
Agreed. Especially if push is working. A much longer polling makes sense to
me. 10 minutes seems fine.

**Comment by nicpottier** [comment]
Norbert once you get this built with the other PR if you can share a signed APK we can test over the next few days would be great.

**Comment by nicpottier** [comment]
Norbert once you get this built with the other PR if you can share a signed APK we can test over the next few days would be great.

**Pull request merged**